src/components/SlateEditor/plugins/DND/onDrop.ts

```diff
@@ -6,10 +6,10 @@ import { getTopNode } from './utils';
 
 const onDrop = (editor: Editor): DragEventHandler<HTMLDivElement> => event => {
   const data = event.dataTransfer;
-
   if (data.getData('application/x-slate-fragment')) {
     // Prevent slate from merging current event with the previous by inserting an empty undo.
-    editor.history.undos.push([]);
+    // Reason: Undo after drag and drop causes both drag and previous action to be undone.
+    editor.history.undos.push({ operations: [], selectionBefore: null });
```

Comment:
Her legger du bare til en tom operasjon, slik at den blir fjerna først? E

Reply:
Slate lager automatisk en historikk med hendelser som kan undoes og redoes, men i noen tilfeller finner den ut at to hendelser skal slås sammen. Problemet vi har hatt med dette er at skriving av tekst etterfulgt av drag-n-drop av noe blir merget til et event. Derfor legger vi heller til et tomt event som drag-n-drop merges inn med.

En mulig sideeffekt av dette er at det faktisk blir værende en tom operasjon noen ganger og at ctrl+z ikke fører til noe på akkurat den operasjonen